connector_amazon/models/amazon_backend.py

```diff
@@ -46,8 +54,12 @@ class AmazonBackend(models.Model):
             ('mws.amazonservices.in', 'India (IN)'),
             ('mws.amazonservices.com.cn', 'China (CN)'),
             ('mws.amazonservices.jp', 'Japan (JP)'),
-            ],
-        required=True)
+        ], required=True)
+    encoding = fields.Selection(
+        selection=[
+            ('ISO-8859-15', 'ISO-8859-15'),
+            ('UTF-8', 'UTF-8'),
```

Comment:
@paulius-sladkevicius do you know what are encodings all over the world ?
ISO-8859-15 sufficent ?